src/cdk/a11y/aria-describer.ts

```diff
@@ -147,8 +146,10 @@ export class AriaDescriber {
 
   /** Deletes the global messages container. */
   private _deleteMessagesContainer() {
-    this._document.body.removeChild(messagesContainer!);
-    messagesContainer = null;
+    if (messagesContainer && messagesContainer.parentNode) {
```

Comment:
```
    if (messagesContainer && messagesContainer.parentNode) {
      messagesContainer.parentNode.removeChild(messagesContainer);
      messagesContainer = null;
    }
```
if messagesContainer.parentNode is not defined and messagesContainer is not null object, 
messagesContainer cannot be null. 

```
    if (messagesContainer) {
      if (messagesContainer.parentNode) {
         messagesContainer.parentNode.removeChild(messagesContainer);
      }
      messagesContainer = null;
    }
```